### PR TITLE
add a console app

### PR DESCRIPTION
**Pull request by ElizabethOkerio**
This PR is a partial fulfillment of the "OCS CLI tool" work item. The PR adds a console app and a bit of restructuring of the project folders' structure. 

Created two folders in the src folder. <code>ODataConnectedService</code> and <code>ODataConnectedServiceCLI</code>. I moved all the files that were in the src folder to the <code>ODataConnectedService</code> folder. On the other hand, the <code>ODataConnectedServiceCLI</code> folder will have all the CLI tool files.

I also added a solution file for the console app in the root folder. <code>ODataConnectedServiceCLI.sln </code>
I also added a generate command with some of the options that will be passed to it and a class library that will have some of the core classes or functions/methods.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/OData/ODataConnectedService?pullRequest=232) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/OData/ODataConnectedService?pullRequest=232) before we can accept your contribution.<br/><br/>:x: ElizabethOkerio [sign now](https://cla.dotnetfoundation.org/OData/ODataConnectedService?pullRequest=232)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/OData/ODataConnectedService?pullRequest=232) it.</sub>

File: src/ODataConnectedServiceCli/ODataConnectedServiceCLI.csproj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (gathogojr) [inline src/ODataConnectedServiceCli/ODataConnectedServiceCLI.csproj:1]: @ElizabethOkerio Is this file needed? Or why do we have both _src/ODataConnectedServiceCLI/ODataConnectedServiceCLI.csproj_ and _src/ODataConnectedServiceCli/ODataConnectedServiceCLI.csproj_?

File: src/ODataConnectedServiceCli/Program.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
+﻿using System.Reflection;
```
Comment (gathogojr) [inline src/ODataConnectedServiceCli/Program.cs:1]: Can we settle for one naming? Some place I see _src/ODataConnectedServiceCli_ and others _src/ODataConnectedServiceCLI_
